watchdog/watchdog.go

```diff
@@ -46,15 +70,19 @@ func Watch(programs []Program, restartDelay time.Duration, log Log) error {
 	return nil
 }
 
+func StopMatching(path string, ospid int, log Log) {
```

Comment:
you can make this a private method

`func stopMatching(path string, exceptPid int, log Log) `